qiskit/schemas/backend_configuration_schema.json

```diff
@@ -255,7 +255,68 @@
                                 "type": "array",
                                 "minItems": 0,
                                 "description": "A list of available parametric pulse shapes",
-                                "items": {"type": "string"}}
+                                "items": {"type": "string"}},
+                            "qubit_channel_map": {
```

Comment:
This is no longer necessary as this information will be captured in channel_config.